### ISSUE TITLE
Samples tiny window and full screen not working

### ISSUE THREAD

**Issue by WildBillKelso** [floooh/sokol-samples#74]
I can get the emscripten samples to run, but they're in a tiny window, and the fullscreen doesn't work (Windows 10).

How can you get it to start either using all the browser area (without the powered by emscripten sillyness) or fullscreen?

![Sokol_Emscripten Windows](https://user-images.githubusercontent.com/2146928/99751767-e173b780-2ab0-11eb-8860-5af63781f8b2.png)

**Comment by frink** [comment]
You have to provide the HTML template for this...

**On a side note:** Manually compiling with Emscripten is very messy without fips. I'm working on documenting how to do that. But right now I've got 20+ flags that have to be set to get the same things as fips.

**Comment by floooh** [comment]
Yes, you need to "link" with a separate shell.html which looks more or less like this:

https://github.com/floooh/sokol-samples/blob/master/webpage/shell.html

Search for the ```--shell-file``` option here:

https://emscripten.org/docs/tools_reference/emcc.html

**Comment by WildBillKelso** [comment]
I think I found the issue.  The emscripten build instructions suggest to use webgl2-emsc-make-release.

In the folder fips-files\configs, this config doesn't exist (for me).

I was able to get it to work using another config in the configs folder: fips set config webgl2-emsc-ninja-release

This appears to pick up the shell.html in the \webpage folder.

Thanks for pointing me in the right direction!
